tools/topology/topology2/platform/intel/ptl.conf

```diff
@@ -0,0 +1,6 @@
+# PTL-specific variable definitions
+Define {
+	DMIC_DRIVER_VERSION		5
+	SSP_BLOB_VERSION		0x300
+	NUM_HDMIS			3
+}
```

Comment:
This is duplicated in #9306, just a heads up.
cc: @singalsu

Reply:
Yep, same definitions to enable build of PR. Rebase will fix this whichever PR is merged first.